Translation corrections (#255)
07th-mod/higurashi-console-arcs: Update/yoig_005.txt

//r自分から危険に飛び込むのはごめんとばかりに、乙部は尻込みする。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "自分から危険に飛び込むのはごめんとばかりに、乙部は尻込みする。",

> 			NULL, "Otobe hesitated as if to say he wouldn't dare plunge into the danger headfirst.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Otobe hesitated, as if to say he wouldn't dare plunge into the danger headfirst.", Line_Normal);
```

	ClearMessage();

//r…そのくせに、はっきり『嫌だ』と言えないところが余計に癇に障るんだけどな。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…そのくせに、はっきり『嫌だ』と言えないところが余計に癇に障るんだけどな。",

> 			NULL, "...Though the fact he couldn't come right out and say \"I don't want to,\" in spite of that really got on my nerves.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...But despite that, the fact that he couldn't come right out and say \"I don't want to\" really got on my nerves.", Line_Normal);
```
